src/main/java/com/elastic/support/util/LocalSystem.java

```diff
@@ -86,6 +86,38 @@ public void copyLogs(List<String> entries, String logDir, String targetDir) {
         }
     }
 
+    /**
+    * On this function we will try to collect the journalctl logs, 
+    * Some services as Kibana installed with the RPM package will give the access to the logs using the journalctl command
+    *
+    * @param  serviceName
+    * @param  targetDir
+    */
+    @Override
+    public void copyLogsFromJournalctl(String serviceName, String targetDir) {
+
+        String tempDir = "templogs";
+        String mkdir = "mkdir templogs";
+        String journalctl = " journalctl -u {{SERVICE}} > '{{TEMP}}/{{SERVICE}}.log'";
```

Comment:
Does this need a leading space for the `runCommand`?

```suggestion
        String journalctl = "journalctl -u {{SERVICE}} > '{{TEMP}}/{{SERVICE}}.log'";
```